### ISSUE TITLE
Simplify process to check a model's results

### ISSUE THREAD

**Issue by Currie32** [VanLifeInc/models#4]
It's currently a bit of a hassle to check the results of a model. Script(s) should be written to allow anyone to play around with a trained model to understand its predictive ability.

**Comment by oliverfoster27** [comment]
Good idea. I suggest creating an object class called (say) analyzer and pass the trained model object to it. Once its been instantiated & initialized we can write methods on the analyzer object to call depending on what aspect of the model you want to look at. This will prevent data leakage by avoiding running methods directly on the trained model object.

**Comment by Currie32** [comment]
This issue was resolved by https://github.com/VanLifeInc/models/pull/5
